elizhahn/niche-audience
# Feature/landing page

## What does this PR do (summary of changes)?
- The App and Home components were built
- The Home component will be the landing page, and hosts the title, input for a user to enter their name, and a get started button
- React router was installed and setup for use. When a user clicks on the "get started" button, they will be routed to the dashboard and their name should display on the dashboard view
- Normalize was set up, and Sass, initial styling for the Home page was added
## How should it be tested?
- Visit the home page and fill out the name input and click get started
- You should be routed to a new view, and the name should be displaying
## Future Iterations:
- Error handling if the input is not filled out
